clog/loggers.py

```diff
@@ -169,7 +172,7 @@ def _log_line_no_size_limit(self, stream, line):
            Since this method is called in log_line, the line should be in utf-8 format and
            less than MAX_LINE_SIZE_IN_BYTES already. We don't limit traceback size.
         """
-        with self.__lock:
+        with self.__lock, self.metrics.sampled_request():
```

Comment:
This is not py26 compatible I'm afraid. We have py26 compatibility in the setup.py.
Although apparently we don't run tests for it. We can probably just drop py26 support.